src/features/vote/useTableValues.ts

```diff
@@ -0,0 +1,137 @@
+import { useState, useEffect } from "react";
+import { EncryptedVote } from "web3/get/queryEncryptedVotesEvents";
+import { PendingRequest } from "web3/get/queryGetPendingRequests";
+import { DateTime } from "luxon";
+import { VoteRevealed } from "web3/get/queryVotesRevealedEvents";
+import { ethers } from "ethers";
+import { PostRevealData } from "web3/post/revealVotes";
+import web3 from "web3";
+
+interface TableValue {
+  ancillaryData: string;
+  identifier: string;
+  vote: string;
+  revealed: boolean;
+  ancHex: string;
+  timestamp: string;
+}
+
+export default function useTableValues(
```

Comment:
was this just pulled out of a component? if so what was changed?

Reply:
Basically, the table values in both Commit and Reveal are super similar. So combined them into a hook for DRY.